javascript/ql/test/library-tests/CapturedNodes/method-calls.js

```diff
@@ -0,0 +1,57 @@
+(function() {
```

Comment:
Could you annotate the various calls in this file with a comment explaining what we can and cannot infer about them? I know that this information is implicit in the `.expected` files, but it's hard to compare the two during review.